### PR TITLE
feat(admin, doctor, nurse, patient): add input validation

### PR DESCRIPTION
**Pull request by dhichii**
This validates the input format and min length of creating new or update existing data.
### Admin, Doctor, Nurse
- Create new user
### Patient
- Create new patient
- Update patient

### CODE REVIEW

**Comment by thisham** [comment]
Is this validations doesn't contain validation error descriptions yet, right?

File: src/app/doctor/handlers/handlers.go [hunk 1]
```diff
@@ -24,6 +26,26 @@ func (h *Handler) CreateDoctorHandler(c echo.Context) error {
...
+
+		if strings.Contains(err.Error(), "password") {
+			return utils.CreateEchoResponse(
+				c,
+				http.StatusBadRequest,
+				response.ErrorResponse{
+					Reason: "password must have at least 8 characters",
+				},
+			)
+		}
```
Comment (thisham) [inline src/app/doctor/handlers/handlers.go:46]: Why the error reasons not declared in a variable then return the echo response once?

**Review by thisham — COMMENTED** [review]
How do you deals with a multiple field errors?

File: src/app/doctor/handlers/handlers.go (hunk 1 above)
Comment (dhichii) [reply src/app/doctor/handlers/handlers.go]: done https://github.com/ALTERA-CAPSTON-41/backend/pull/65/commits/5b09e3601784691dfab73275a3470fe55137059d

**Review by thisham — APPROVED** [review]
LGTM!

**Pull request merged**